takwimu/takwimu_ui/src/components/Faqs/index.js

```diff
@@ -5,24 +5,33 @@ import { withStyles, Grid, Typography } from '@material-ui/core';
 import ContentSection from '../ContentSection';
 import Faq from './Faq';
 
-const styles = () => ({
+const styles = theme => ({
   root: {},
-  contentGrid: { paddingTop: '2rem', paddingBottom: '2rem' }
+  contentGrid: {
+    paddingTop: '2rem',
+    paddingBottom: '2rem',
+    '& a': {
+      color: theme.palette.primary.main
+    }
+  }
 });
 
-function Faqs({ classes, faqs, ...props }) {
+function Faqs({ classes, faqs: { value: faqs }, ...props }) {
+  if (!faqs) {
+    return null;
+  }
+
   return (
     <ContentSection
-      title="Frequently Asked Questions"
+      title={faqs.title}
```

Comment:
maybe toplevel `faqs` ->` faqSection`

so `faqSection: { title, descriptions, faqs }`

Reply:
Everything will then be a section e.g. `services` will also need to be `servicesSection: { title, sections }`. If we follow Wagtail format, i.e `block: { value: block_value }` I think it's easy enough to know and hence the choice to just name these `faqs`, `services`, etc.